calcom/cal.com
# fix: smtp send email fix for self hosted instance

## What does this PR do?

Bug fix for sending email for a self hosted instance

Fixes #10592 

## Type of change




## How should this be tested?



Create a self hosted instance and book any meeting and check if the emails are being sent correctly

## Mandatory Tasks
